packages/vscode-extension/messages.json

```diff
@@ -10,6 +10,9 @@
 	"ANSWERS_NO": "No",
 	"ANSWERS_YES": "Yes",
 
+	"BUTTONS_RESTART_NOW": "Restart Now",
```

Comment:
Why "BUTTONS" instead of "BUTTON" ?

Reply:
It comes from the TranslationKeys.ts were we have other "groups" of keys, like another one called ANSWERS.